grpc/src/main/java/com/linecorp/armeria/client/grpc/GrpcClientBuilder.java

```diff
@@ -287,6 +289,15 @@ public GrpcClientBuilder enableUnsafeWrapResponseBuffers(boolean enableUnsafeWra
         return option(UNSAFE_WRAP_RESPONSE_BUFFERS.newValue(enableUnsafeWrapResponseBuffers));
     }
 
+    /**
+     * Sets whether to respect the marshaller specified in gRPC {@link MethodDescriptor}
+     * If not set, will use the default(false), which use more efficient way that reduce copy operation.
+     */
+    @UnstableApi
+    public GrpcClientBuilder useMethodMarshaller(boolean useMethodMarshaller) {
+        return option(USE_METHOD_MARSHALLER.newValue(useMethodMarshaller));
+    }
```

Comment:
It looks to me that this method is not called in this pull request. Could you add more test cases to ensure this feature actually works as expected when a user configures the client using this method?

Reply:
Okay, I'll check and add test case for it.

Reply:
Thanks for letting me know about missing testcase. 
I've added test for this method.